cmd/create/operatorroles/by_prefix.go

```diff
@@ -54,7 +54,7 @@ func handleOperatorRolesPrefixOptions(r *rosa.Runtime, cmd *cobra.Command) {
 		os.Exit(1)
 	}
 	args.hostedCp = isHostedCP
-	args.installerRoleArn = interactive.GetInstallerRoleArn(r, cmd, args.installerRoleArn, "")
+	args.installerRoleArn = interactive.GetInstallerRoleArn(r, cmd, args.installerRoleArn, "", false)
```

Comment:
@gdbranco I assume we should pass here the value `args.hostedCp`, can you confirm?

Reply:
Yes, for this one this is correct